### PR TITLE
Better setup

### PR DESCRIPTION
**Pull request by chen-robert**
Added `install/config.sh` to automatically set up all configuration after cloning. 

Also cleaned up some minor deployment issues throughout.

### CODE REVIEW

**Comment by Arinerron** [comment]
Getting this issue from the last line of the config script:
```
Creating rctf_postgres_1 ... done
yarn run v1.21.1
warning Skipping preferred cache folder "/.cache/yarn" because it is not writable.
warning Selected the next writable cache folder in the list, will be "/tmp/.yarn-cache-999".
$ node-pg-migrate --database-url-var=RCTF_DATABASE_URL up
warning Cannot find a suitable global folder. Tried these: "/usr/local, /.yarn"
/bin/sh: 1: node-pg-migrate: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

File: README.md [hunk 1]
```diff
@@ -28,7 +28,14 @@ The automatic installation script works on Debian-based distributions and Arch L
 
 #### Manual
 
-For instructions on manual installations and deployments, see the [wiki page](https://github.com/redpwn/rctf/wiki/Manual-Deployment).
```
Comment (Arinerron) [inline README.md]: y remove this? I moved it to the wiki cause most people won't manual install

File: install/config.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cp .env.example .env
+cp -r .rdeploy.example .rdeploy
+cp config/client.js.example config/client.js
+
+mkdir -p data
+mkdir -p data/rctf-postgres
+mkdir -p data/rctf-redis
```
Comment (Arinerron) [inline install/config.sh]: There are some (somewhat nitpicky) issues with these lines, but I'll fix in a future PR.

File: README.md (hunk 1 above)
Comment (chen-robert) [reply README.md]: The manual install is short enough to include in the README now I think, no more copying around of config files. It's also useful for installing on Amazon AMIs for example, I think get.rctf doesn't support that right now.

**Comment by chen-robert** [comment]
@Arinerron should have been resolved in https://github.com/redpwn/rctf/commit/4545dfb82e444b1f108d38db58d10637f128f40e#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R29? Can you confirm if this still appears on master

**Pull request merged**